CHANGELOG.md

```diff
@@ -2,86 +2,63 @@
 
 ## Release (1.3.0)
 
-### Ajouts généraux 
-### Commandes ajoutées
+In this release, we decided to use English as the main language.
 
-### Corrections de bogues
+### additions
 
-- Correction du `SkipTitle` en cas de "test de combat" (@Jauke)
+### Bug fixes
+- Fix `SkipTitle` in the "battle-test-case" (@Jauke)
 
 ## Release (1.2.0)
 
-### Ajouts généraux
-
-* Gestion des Intervalles dans les variables locales : `SV[1, 2, 1..18] = 10` (par exemple) par @xvw
-
-### Commandes ajoutées
-
-* `camera_scrolling?` (@xvw, reporté par @Zer0Zer0x)
-* `windowskin_tone` et `windowskin_opacity` (@xvw, reporté par @YendaHusk)
-* `event_tone` et `player_tone` (@xvw, reporté par @Zer0Zer0x)
-* `event_character_name`, (@BastienDuplessier)
-* `event_character_index`, (@BastienDuplessier)
-* `event_change_character`, (@BastienDuplessier)
-*    - `menu_enabled?|disabled?` 
-     - `save_enabled?|disabled?`
-     - `formation_enabled?|disabled?`
-     - `encounter_enabled?|disabled?`
-     (@xvw, reporté par Zer0Zer0x)
-* `current_event_id` et `me` (déplacement de EvEx vers Command), (@BastienDuplessier)
-* `use_item`, (@BastienDuplessier)
-* `test_item`, (@BastienDuplessier)
-
-### Corrections de bogues
-
-* Correction de `me` dans les trajectoire d'événement (@xvw, reporté par @Zer0Zer0x)
-* Correction des commandes pour flasher les cases (@xvw)
-* Correction de la commande `picture_zoom` (@xvw)
-* Correction de la commande `event_priority` (@xvw, reporté par @Zer0Zer0x)
-* Correction de la commande `event_trigger` (@xvw)
+### additions
+* Add range support in the syntax shortcuts for variables : `SV[1, 2, 1..18] = 10` (for example) (@xvw)
+* Add command `camera_scrolling?` (@xvw, reported by @Zer0Zer0x)
+* Add commandsn `windowskin_tone` and `windowskin_opacity` (@xvw, reported by @YendaHusk)
+* Add commands `event_tone` and `player_tone` (@xvw, reported by @Zer0Zer0x)
+* Add command `event_character_name` (@BastienDuplessier)
+* Add command `event_character_index` (@BastienDuplessier)
+* Add command `event_change_character` (@BastienDuplessier)
+*    - Add command `menu_enabled?|disabled?` 
```

Comment:
The sub bullets' symbol (`-`) should be removed: the sublist should be flatten.
Could you remove the **trailing whitespace** as well ? [#sameplayershootagain]